packages/app/src/systems/SignUp/routes.tsx

```diff
@@ -1,11 +1,20 @@
-import { Navigate, Route } from "react-router-dom";
+import { Navigate, Outlet, Route } from "react-router-dom";
 
-import { WelcomeScreen } from "./pages";
+import { CreateWallet, RecoverWallet, WelcomeScreen } from "./pages";
 
-import { Pages } from "~/systems/Core";
+import { Pages, PublicRoute } from "~/systems/Core";
+
+const wrapper = (
+  <PublicRoute>
+    <Outlet />
+  </PublicRoute>
+);
 
 export const signUpRoutes = (
-  <Route path={`${Pages.signUp}/*`} element={<WelcomeScreen />}>
-    <Route index element={<Navigate to={Pages.signUp} />} />
+  <Route path={`${Pages.signUp}/*`} element={wrapper}>
+    <Route index element={<Navigate to={Pages["signUp.welcome"]} />} />
+    <Route path={Pages["signUp.welcome"]} element={<WelcomeScreen />} />
```

Comment:
I think we should change the `Pages["signUp.welcome"]` to `Pages.signUpWelcome`

Reply:
✅